JavaImportTool/src/javaImportTool/ASTConverter.java

```diff
@@ -654,17 +653,19 @@ public void processTypeParameters(List<TypeParameter> params) throws ConversionE
 		}
 	}
 	
-	public void processParameters(List<SingleVariableDeclaration> params) throws ConversionException
+	public void processParameters(List<VariableDeclaration> params) throws ConversionException
 	{
 		int i = 0;
-		for(SingleVariableDeclaration arg : params)
+		for(VariableDeclaration arg : params)
 		{
 			Node a = containers.peek().child("arguments").add(new Node(null,"FormalArgument", i++));
 			a.setSymbol(arg.getName().getIdentifier());
 			setModifiersAndAnnotations(a, Modifier.PUBLIC, null);
 			
-			Node type = typeExpression(arg.getType(), "typeExpression");
-			a.setChild("typeExpression", addExtraDimensions(type,arg.getExtraDimensions()));
+			if (arg instanceof SingleVariableDeclaration) {
```

Comment:
Then it is a [VariableDeclarationFragment](http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2Fdom%2FVariableDeclarationFragment.html) which doesn't have a type. Maybe we should then set the type to an EmptyExpression?